rental.rb

```diff
@@ -0,0 +1,9 @@
+class Rental
+  attr_accessor :date, :book, :person
+
+  def initialize(date, book, person)
+    @date = date
+    @book = book
+    @person = person
```

Comment:
- You are yet to implement the `belongs-to` relationship between the `rental` class and the `book` and `person` class. i.e 
```ruby
book.rentals << self
```
```ruby
person.rentals << self
```